spec/turn_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'rspec'
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+require './lib/turn'
+require './lib/game'
+
+RSpec.describe Deck do
+  it 'initialize' do
+    card1 = Card.new(:heart, 'Jack', 11)
+    card2 = Card.new(:heart, '10', 10)
+    card3 = Card.new(:heart, '9', 9)
+    card4 = Card.new(:diamond, 'Jack', 11)
+    card5 = Card.new(:heart, '8', 8)
+    card6 = Card.new(:diamond, 'Queen', 12)
+    card7 = Card.new(:heart, '3', 3)
+    card8 = Card.new(:diamond, '2', 2) 
+    deck1 = Deck.new([card1, card2, card5, card8])  
+    deck2 = Deck.new([card3, card4, card6, card7]) 
+    player1 = Player.new('Megan', deck1)
+    player2 = Player.new('Aurora', deck2)
+    expect(player1.name).to eq('Megan')
+
+    turn = Turn.new(player1, player2)     
+
+  end 
```

Comment:
The `it` block wants to test `Turn` initialization but we, unfortunately, do not have an expectation in here that does that. `expect(player1.name).to eq('Megan')` has already been tested in the `Player` spec so we are ok not testing it again here.

The assertions/expectations I would have liked to see here are:

```ruby
expect(turn).to be_an_instance_of(Turn)
expect(turn.player1).to eq(player1)
expect(turn.player2).to eq(player2)
```